packages/packages.yaml.tmpl

```diff
@@ -166,7 +164,26 @@ components:
             - "{{ .Git.ref }}"
         steps:
           release:
-            - script: make cdc dm-master-with-webui dm-worker dmctl dm-syncer
+            - os: darwin
+              description: install nodejs toolchain.
+              script: |
+                NODE_VERSION="16.11.12"
+                NVM_VERSION=v0.39.5
+                NVM_DIR=/usr/local/nvm
+                mkdir -p $NVM_DIR
+
+                if [ ! -s "$NVM_DIR/nvm.sh" ]; then
+                  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/${NVM_VERSION}/install.sh | bash
+                fi
+                . $NVM_DIR/nvm.sh
+                nvm install ${NODE_VERSION}
+                nvm use v${NODE_VERSION}
+                nvm alias default v${NODE_VERSION}
+
+                node --version && npm --version
+                npm install -g yarn
```

Comment:
should toolchain installing script be here? install toolchain every time seems slow. How about just write the dependencies?
Besides, there is no llvm/gcc installing script for tikv/tiflash

Reply:
It's tiflow!

Reply:
tikv, tiflash is not ready.

nvm install will skip if the version installed, so do not worry about it.